examples/Client/Cryptography/Cryptography.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net8.0</TargetFramework>
```

Comment:
I don't know if there's any official guidance, but it looks like most of our examples target `net6`.

Reply:
I switched to .NET 8 after seeing your [PR for the version bump](https://github.com/dapr/dotnet-sdk/pull/1188) get merged for the SDK itself. Seems if the SDK is using the latest, the examples probably should too, but that can be another PR.

Reply:
The NuGet packages now ship *both* .NET 6 *and* .NET 8 targets (as there are some minor optimizations allowed for .NET 7/8).  Given both are LTS versions, I can see valid arguments either way for the sample target versions.  I'd say stick with .NET 6 for now, for the sake of consistency with the others.  In November, when .NET LTS ends for .NET 6, it'd make sense to move the samples as a whole to .NET 8.

Reply:
Downgraded to .NET 6